02-express-tutorial/Public/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>Hello Everyone.</p>
+<p>My name is Melanie.</p>
+
+<button id="fetchProductsButton">Fetch Products</button>
+    <div id="productDisplay"></div>
+
+
+<script>
+
+document.getElementById('fetchDataButton').addEventListener('click', () => {
+    fetch('/api/v1/products')
+    .then((response) => response.json())
+    .then((data) => {
+        const productListDiv = document.getElementById('productList');
+        productListDiv.innerHTML = '';
+        data.forEach((product) => {
+        const productInfo = document.createElement('div');
+        productInfo.innerHTML = `<strong>${product.name}</strong>: $${product.price}`;
+        productListDiv.appendChild(productInfo);
+        });
+    })
+    .catch((error) => {
```

Comment:
Nitpick (doesn't require changes):
If there's ever an error in the API response, we want to make sure that error is communicated to the user. I'd recommend that we handle the error and display a user friendly error message that states "Unable to load products, please try again later."